yaksh/views.py

```diff
@@ -1071,9 +1072,24 @@ def show_all_questions(request):
             else:
                 context["msg"] = "Please select atleast one question to test"
 
+        if request.POST.get('question_tags'):
+            question_tags = request.POST.getlist("question_tags")
+            search_tags = []
+            for tags in question_tags:
+                search_tags.extend(re.split('[; |, |\*|\n]',tags))
```

Comment:
Can this be pushed to client-side JS and have the JS send a JSON string? We can then convert the JSON to a dict/list in a easier fashion.